src/display/api.js

```diff
@@ -753,6 +753,15 @@ class PDFDocumentProxy {
     return this._transport.getJavaScript();
   }
 
+  /**
+   * @returns {Promise<Object | null>} A promise that is resolved with
+   *   an {Object} with the JavaScript actions, or `null`
+   *   if no JavaScript exists.
+   */
+  getJSActions() {
```

Comment:
Given how similar this is to the existing `getJavaScript` method just above, there should probably be some kind of (brief) explanation of the differences here.